tests/btrfs-progs/install.pm

```diff
@@ -48,12 +49,17 @@ sub run {
     assert_script_run('git clone -q --depth 1 ' . GIT_URL, timeout => 360);
     assert_script_run 'cd btrfs-progs';
     assert_script_run './autogen.sh';
-    assert_script_run './configure --disable-documentation --disable-convert --disable-zstd \
---disable-programs --disable-shared --disable-static --disable-python';
-    assert_script_run 'make testsuite', timeout => 300;
+    assert_script_run './configure --disable-documentation --disable-zstd --disable-python';
+    assert_script_run 'make', timeout => 600;
+    assert_script_run 'make testsuite';
     assert_script_run 'mkdir -p ' . INST_DIR;
     assert_script_run 'tar zxf tests/btrfs-progs-tests.tar.gz -C ' . INST_DIR;
+    assert_script_run 'cp btrfs btrfs-convert btrfs-find-root btrfs-image btrfs-select-super btrfstune /sbin/';
```

Comment:
Why not convert this to a bash script, put it into the data directory? and pass necessary arguments instead?

Reply:
Thanks your suggestion. Actually test scripts will find the path with _dirname `which btrfs`_, so I can copy all binary into the data directory and change the _PATH_ value.
But why I need a bash script?

Reply:
> Thanks your suggestion. Actually test scripts will find the path with _dirname `which btrfs`_, so I can copy all binary into the data directory and change the _PATH_ value.
> But why I need a bash script?

The suggestion comes more because what you are doing is basically a full-on console test, and you are not using other features that check the screen... so you might aswell put everything in bash and report the results for what's actually suposed to be reported (if needed):

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/58e0264157511bbe11a50c938ef026c48a6edeec/tests/console/sqlite3.pm#L36

Reply:
BTW, what's benefit to copy these into data directory? For the minimal effect on system under test?

Reply:
@lansuse https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/bf9d9054b1e46b0fdbc7b8fdd694066f4f9697c3/tests/yomi/start_qemu.pm#L36

However my suggestion it's just a suggestion :) you can take it or leave it ^^

Reply:
> BTW, what's benefit to copy these into data directory? For the minimal effect on system under test?

There will be less probability to be hit by external noise (Mistyping issues), if you put it into the data directory, and just call the script... it's a lot less things that are being typed... 

openQA has a limit, that if the amount of characters is less than 80 (iirc) it will type, otherwise it will create a script on the fly, download it, and then execute it :).

Reply:
Thank you. This looks pretty good. I'll double check it on my local machine:)

Reply:
> @lansuse
> 
> https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/bf9d9054b1e46b0fdbc7b8fdd694066f4f9697c3/tests/yomi/start_qemu.pm#L36
> 
> However my suggestion it's just a suggestion :) you can take it or leave it ^^

Thanks, modified.

Reply:
> @lansuse
> 
> https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/bf9d9054b1e46b0fdbc7b8fdd694066f4f9697c3/tests/yomi/start_qemu.pm#L36
> 
> However my suggestion it's just a suggestion :) you can take it or leave it ^^

Thanks, modified.